src/tests/ftest/util/dfuse_utils.py

```diff
@@ -52,7 +53,7 @@ def set_dfuse_pool_params(self, pool, display=True):
             pool (TestPool): DAOS test pool object
             display (bool, optional): print updated params. Defaults to True.
         """
-        self.puuid.update(pool.uuid, "puuid" if display else None)
+        self.puuid.update(pool.label, "puuid" if display else None)
```

Comment:
The pool class allows switching between uuid/label. We should do this instead, which uses the label by default:
```suggestion
        self.puuid.update(pool.identifier, "puuid" if display else None)
```